config/webpack.dev-stage.config.js

```diff
@@ -160,6 +160,7 @@ module.exports = merge(commonConfig, {
       chunks: ['app'],
       FAVICON_URL: process.env.FAVICON_URL || null,
       OPTIMIZELY_PROJECT_ID: process.env.OPTIMIZELY_PROJECT_ID || null,
+      ROBOT_CONTENT_TAG: process.env.ROBOT_CONTENT_TAG || null, // Configures "content" value of a "robots" meta tag
```

Comment:
nit: perhaps `META_ROBOTS_CONTENT_TAG` or `META_ROBOTS_TAG_CONTENT_ATTR` (i.e., consider adding `META_` prefix since its related to a `<meta>` tag, with a `content` attribute).

Reply:
I like `META_TAG_ROBOTS_CONTENT_ATTR` for clarity. I will update. Thanks!